### PR TITLE
Fix VBOX-Loader

### PR DESCRIPTION
**Pull request by cecinestpasunepipe**
Part of the VBOX-loader fix

(DIS-1430)

### CODE REVIEW

**Review by Schamper — CHANGES_REQUESTED** [review]

File: dissect/target/loaders/vbox.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def detect(path):
         return path.suffix.lower() == ".vbox"
 
     def map(self, target):
-        for disk, _ in self.vbox.snapshots():
+        for disk in self.vbox.disks():
             parent = self.open(disk)
             target.disks.add(VdiContainer(parent))
```
[the remark this answers is not in the document]
Comment (cecinestpasunepipe) [reply dissect/target/loaders/vbox.py:23]: For some reason, if I do that, it raises an error and the vdi is not recognized anymore. I will look into this...

[the remark this answers is not in the document]
Comment (cecinestpasunepipe) [reply dissect/target/loaders/vbox.py]: Adjusted, seems that VDI container was using the wrong magic number in detect()?

[the remark this answers is not in the document]
Comment (Schamper) [reply dissect/target/loaders/vbox.py]: Ah interesting, correct magic number, wrong type :)

**Review by Schamper — DISMISSED** [review]

File: tests/test_loaders_vbox.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from unittest.mock import call, patch
+
+from dissect.target.loaders.vbox import VboxLoader
+
+
+@patch("dissect.hypervisor.vdi")
+@patch("dissect.hypervisor.vdi.Vbox")
```
Comment (Schamper) [inline tests/test_loaders_vbox.py:7]:
```suggestion
@patch("dissect.hypervisor.descriptor.vbox.VBox")
```

**Review by Schamper — CHANGES_REQUESTED** [review]

**Review by Schamper — APPROVED** [review]

**Comment by Schamper** [comment]
I can't quite comment on it, but can you rename the class to `VBoxLoader` so it's more consistent with the other rename?

**Review by Schamper — APPROVED** [review]

**Pull request merged**